src/applications/claims-status/utils/helpers.js

```diff
@@ -21,6 +21,19 @@ const phaseMap = {
   8: 'Complete',
 };
 
+const lighthouseStatuses = [
```

Comment:
I'm wondering if it would be better to use the expanded statuses we get from LH instead of the condensed ones. Thoughts?